notebooks/chapter03/section03_tokenizer.py

```
@@ -0,0 +1,408 @@
+
+
+# =============================================================================
+# 3.2.2 Tokenizerの入力処理
+# =============================================================================
+
+
+def add_special_tokens(tokens, max_length):
+    """
+    特殊トークンを追加する関数
+    """
+    tokens = ["[CLS]"] + tokens + ["[SEP]"]
```

Ruff (0.16.1)

[warning] Consider `["[CLS]", *tokens, "[SEP]"]` instead of concatenation

Replace with `["[CLS]", *tokens, "[SEP]"]`

(RUF005)